robot/rover/MobilePlatform/MobilePlatform.ino

```diff
@@ -1,8 +1,19 @@
 
 #include "Navigation.h"
-
 #include <SoftwareSerial.h>
 #include <Servo.h>
+#include "ArduinoBlue.h"
+
+// TODO: Change to include instead of INCLUDES when make builds
+#include "includes/PinSetup.h"
+#include "../internal_comms/include/CommandCenter.h"
+#include "../internal_comms/include/Serial.h"
+#include "includes/commands/WheelsCommandCenter.h"    
```

Comment:
We don't want anymore todos. Either 1) do the todo before the merge 2) make an issue TODOs are forgotten too easily and they perish in the code

Reply:
Made an issue for this. I am pushing my changes for now so that there is something which can be tested with. Changing Includes to Include is a minor issue, which doesn't need to be addressed right now.